_docs/11-app-config.md

```diff
@@ -318,12 +318,38 @@ In addition to the attributes of the `MenuItem`, you can include other attribute
   | Name  | Type   | Description |
   | ----- | ------ | ----------- |
   | route | string | The route the application will navigate when the menu item is clicked |
+  | pathMach | 'full' \| 'prefix'  | Match criteria to mark the menu item as active in case of multiple routes matching |
 
   <span>Example:</span>
```

Comment:
I think it would be nice to add a little explanation of what means each value ('full' and 'prefix') before the examples.